Projects/STM32H735G-DK/Examples/BSP/Src/audio_play.c

```diff
@@ -360,6 +361,8 @@ static uint8_t AudioPlay_GetTouchPosition(void)
     }
   }
 
+  if (touch == touch_previous) return 0;
```

Comment:
Hi @TheseGoTo11,

Thank you for your contribution. In order to adapt the proposed fix, could you update this line code _(line 364)_ as follows: 
```
  if (touch == touch_previous)
  {
    return 0;
  }
```
Thank you again for your contribution.

With regards,

Reply:
OK, I have added the requested change.